_posts/2019-06-20-load-image-data.md

```diff
@@ -1053,7 +1054,8 @@ https://docs.dask.org/en/latest/array-api.html#dask.array.Array.to_zarr )
 method.
 
 ```python
-a.to_zarr("mydata.zarr")
+from numcodecs import Blosc
+a.to_zarr("mydata.zarr", compressor=Blosc(cname='zstd', clevel=3, shuffle=Blosc.BITSHUFFLE))
```

Comment:
I'd probably leave the code as-is here and then note something below about other compressors that one could use and this particular configuration being a good option. WDYT?

Reply:
Sounds good :+1: